src/runtime-rs/src/shim/src/delete.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021 Alibaba Cloud
+// Copyright (c) 2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use protobuf::Message;
+
+use protocols::shim as shim_proto;
+use virtcontainers::Sandbox;
+
+use crate::{Error, Result, ShimArgs};
+
+pub fn do_delete(args: &ShimArgs) {
+    match cleanup(args) {
```

Comment:
I do not much understand your point. In this function, if `cleanup` returns ok it will write result to stdout, otherwise it will write errors to stderr